libs/components/src/lib/text-field/text-field.spec.ts

```diff
@@ -550,6 +550,21 @@ describe('vwc-text-field', () => {
 			expect(iconElement?.getAttribute('name'))
 				.toEqual(iconName);
 		});
+		it('should set class has-icon when icon is set', async function () {
+			element.icon = 'home';
+			await elementUpdated(element);
+			const baseElementHasIconClass = element.shadowRoot?.
+				querySelector('.base')?.classList.contains('has-icon');
+			expect(baseElementHasIconClass).toEqual(true);
+
+		});
+		it('should remove .no-leading class to .base if icon is set', async function () {
+			element.icon = 'home';
+			await elementUpdated(element);
+			const baseElementHasIcon = element.shadowRoot?.
+				querySelector('.base')?.classList.contains('no-leading');
+			expect(baseElementHasIcon).toEqual(false);
```

Comment:
Just updating the wording
```suggestion
		it('should remove .no-leading class from .base if icon is set', async function () {
			element.icon = 'home';
			await elementUpdated(element);
			const baseElementHasNoLeadingClass = element.shadowRoot?.
				querySelector('.base')?.classList.contains('no-leading');
			expect(baseElementHasNoLeadingClass).toEqual(false);
```